package.json

```diff
@@ -29,7 +29,8 @@
     "@types/babel__preset-env": "7.9.1",
     "@types/eslint": "^7.2.4",
     "@types/eslint-plugin-prettier": "^3.1.0",
-    "@types/prettier": "^2.1.2",
+    "@types/lodash": "^4.14.162",
```

Comment:
Can we get rid of `lodash` please?

Reply:
Okay, I will try to find a different solution for that function. Maybe at some point in the future lodash will not be anymore in the deps of deps, so getting rid of it is a good idea after all.

Reply:
done